--- a/glib/gmessages.c	2016-09-12 15:48:48.000000000 +0000
+++ b/glib/gmessages.c	2017-10-23 04:31:45.000000000 +0000
@@ -2061,9 +2061,16 @@
   /* Message was too large, so dump to temporary file
    * and pass an FD to the journal
    */
-  if ((buf_fd = mkostemp (path, O_CLOEXEC|O_RDWR)) < 0)
+//  if ((buf_fd = mkostemp (path, O_CLOEXEC|O_RDWR)) < 0)
+  if ((buf_fd = mkstemp (path)) < 0)
     return -1;
 
+  if (fcntl(buf_fd, FD_CLOEXEC) < 0)
+    {
+      close (buf_fd);
+      return -1;
+    }
+
   if (unlink (path) < 0)
     {
       close (buf_fd);
