--- a/setup.py	2017-12-07 16:00:28.089735529 +1100
+++ b/setup.py	2017-12-07 16:00:50.856827293 +1100
@@ -2,7 +2,7 @@
 major, minor1, minor2, s, tmp = sys.version_info
 if major==2 and minor1<4 or major<2:
     raise SystemExit("""matplotlib and the basemap toolkit require Python 2.4 or later.""")
-from numpy.distutils.core  import setup, Extension
+from setuptools import setup, Extension
 from distutils.util import convert_path
 from distutils import ccompiler, sysconfig
 try:
