--- a/src/tools/qvaluelist.h	2008-01-16 06:09:13.000000000 +1100
+++ b/src/tools/qvaluelist.h	2017-05-11 22:16:52.127047890 +1000
@@ -50,6 +50,7 @@
 #ifndef QT_NO_STL
 #include <iterator>
 #include <list>
+#include <cstddef>
 #endif
 
 //#define QT_CHECK_VALUELIST_RANGE
--- a/src/tools/qvaluevector.h	2008-01-16 06:09:13.000000000 +1100
+++ b/src/tools/qvaluevector.h	2017-05-11 22:17:18.077672942 +1000
@@ -47,6 +47,7 @@
 
 #ifndef QT_NO_STL
 #include <vector>
+#include <cstddef>
 #endif
 
 template <class T>
